src/test/util/setup_common.h

```diff
@@ -82,12 +82,17 @@ struct BasicTestingSetup {
     const fs::path m_path_root;
 };
 
+struct ChainTestingSetup : public BasicTestingSetup {
```

Comment:
This is only used in validation_chainstatemanager_tests.cpp. Rather than adding complexity to the common setup for all tests, can you define this struct inside validation_chainstatemanager_tests.cpp and leave setup_common unchanged?